src/test/java/edu/kis/powp/command/OperateToCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.Job2DriverToDrawerAdapterSpecial;
+
+public class OperateToCommand implements DriverCommand{
+    private int x = 0, y = 0;
+    private final Job2dDriver adapter;
```

Comment:
The name adapter is wrong.
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).